### PR TITLE
Generate tar.gz for mudrod-engine

### PR DESCRIPTION
**Pull request by fgreg**
This is to support MUDROD-81

updating assembly plugin for core to create a tar.gz distribution of the mudrod-engine tool

### CODE REVIEW

**Comment by fgreg** [comment]
I've changed the appassembler to produce a flat 'lib' directory with all jar dependencies. I also updated the assembler plugin to produce tar.gz and zip versions of the mudrod-engine tool.

The generated tar.gz is what will get copied to the JPL mudrod-server to be used for log ingestion.

File: core/src/main/assembly/bin.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lewismc) [inline core/src/main/assembly/bin.xml:1]: Can you please add an Apache license header.

**Review by lewismc — CHANGES_REQUESTED** [review]
This looks good @fgreg if you could add the license header then please merge into master. Thanks.

**Comment by lewismc** [comment]
@fgreg do you have write access to Mudrod?

**Comment by fgreg** [comment]
@lewismc I think I do. Added license header to bin.xml, look good?

**Comment by lewismc** [comment]
ACK @fgreg please merge thanks

**Pull request merged**